test/integration/filters/array.spec.ts

```diff
@@ -104,6 +104,20 @@ describe('filters/array', function () {
       expect(html).toBe('abc')
     })
   })
+  describe('sample', function () {
```

Comment:
This filter relies on randomness so I didn't do anything clever, just ensured the sample size is what we expect.

Reply:
makes sense.